src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+        setArea(Math.pow(radius, 2));
```

Comment:
The area of a circle should be calculated as π * radius^2. Currently, the area is set using Math.pow(radius, 2), which does not include π. Consider using Math.PI * Math.pow(radius, 2) to correctly calculate the area.